File: abc/abc164/abc164_b.cs
// https://atcoder.jp/contests/abc164/submissions/12399772

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var input = Console.ReadLine().Split().Select(int.Parse).ToArray();
        var A = input[0];
        var B = input[1];
        var C = input[2];
        var D = input[3];

        var i = 0;
        while (A > 0 && C > 0)
        {
            if (i % 2 == 1)
            {
                A -= D;
            }
            else
            {
                C -= B;
            }
            i++;
        }
        Console.WriteLine(A <= 0 ? "No" : "Yes");
    }
}

File: abc/abc164/abc164_b.fs
// https://atcoder.jp/contests/abc164/submissions/12399811

let [| A; B; C; D |] = stdin.ReadLine().Split() |> Array.map int

let rec solver a c i =
    if a <= 0 then "No"
    elif c <= 0 then "Yes"
    elif i % 2 = 1 then solver (a - D) c (i + 1)
    else solver a (c - B) (i + 1)

solver A C 0 |> stdout.WriteLine
